magento-research/pwa-studio
# [doc] update useWindowSize docs

## Description

Update doc blocks in `useWindowSize.js` file.

## Related Issue

Closes #1344 

## Verification Steps

1. Navigate to the `pwa-devdocs` directory
2. Create the reference docs: `npm run build-reference-doc-snippets`
3. Verify a `useWindowSize.md` file was created in src/_includes/auto-generated/peregrine/src/hooks/ with doc block content from the `packages/peregrine/src/hooks/useWindowSize.js` source file.

## Screenshots / Screen Captures (if appropriate)

## Checklist:
